src/lib/bootloader/device_map_dialog.rb

```diff
@@ -61,6 +61,11 @@ def controller_loop
             Yast::Popup.Error(_("Device map must contain at least one device"))
             next
           end
+          max_dev = Bootloader::DeviceMap::BIOS_LIMIT
+          if disks.size > max_dev
+            Yast::Popup.Error(_("Device map can have at maximum %i devices") % max_dev)
```

Comment:
NP: I'd use `format` since I found better having _placeholders_ in translations than %i or %s.

```ruby

format(
  # TRANSLATORS: an error message where %{max_devices} is the number of max devices a device can map.
  _("Device map can have at maximum %{max_devices} devices"),
 max_devices: max_dev
)
```

Reply:
btw format is just alias to %...and I agree with you when there are more places as it allows translators to move it freely around. But when there is just one, I prefer short notation.